.github/workflows/deploy-frontend-pr-previews.yml

```diff
@@ -118,8 +118,8 @@ jobs:
       - name: Build and Push Image
         run: |
           cd netmanager/
-          docker build --tag ${{ secrets.REGISTRY_URL }}/${{ secrets.PROJECT_ID }}/pr-previews/netmanager-pr-previews:${{ github.sha }} ./
-          docker push ${{ secrets.REGISTRY_URL }}/${{ secrets.PROJECT_ID }}/pr-previews/netmanager-pr-previews:${{ github.sha }}
+          docker build --tag gcr.io/airqo-250220/pr-previews/netmanager-pr-previews:${{ github.sha }} ./
+          docker push gcr.io/airqo-250220/pr-previews/netmanager-pr-previews:${{ github.sha }}
```

Comment:
Hi @123MwanjeMike , what happened to using ENV variables? Any challenges?

Reply:
Hi @Baalmart, the use of the registry url as an ENV variable was causing several mismatches. It was build and pushing a different image that's why deploys to stage/prod were always having old versions deployed.

Also note that this is not secret information but was rather added as a mechanism to easily change registries with minimal change to the codebase.

Reply:
Thanks @123MwanjeMike, Just have a few queries.

`How is it easy to change registries with minimal change to the codebase??....`
Does one have a raise a PR with over 70+ changes??? if so, isn't very easy for a human error to occur while making the changes??


Trying to understand the logic here.
if `secrets.REGISTRY_URL = gcr.io` and `secrets.PROJECT_ID =  airqo-250220`
how is
`${{ secrets.REGISTRY_URL }}/${{ secrets.PROJECT_ID }}` different from `gcr.io/airqo-250220` ?

Reply:
Hi @NoahNsimbe 
The two are the same actually.
The current issue I am having is that a human error made in the secrets is harder to detect than when it’s made in the visible codebase. That’s why I’m always manually deploying all these applications after changes are not realised.
With regard to the many code changes, I don’t think we shall change our image registry or at least in the foreseeable future.